cit3.0-web/src/components/Page/PowerBi/PowerBi.js

```diff
@@ -174,13 +261,32 @@ export default function PowerBi(props) {
                 }
               },
             ],
+            [
+              "bookmarkApplied",
+              function (event) {
+                console.log("filters", event);
+                window.report.getFilters().then((filters) => {
+                  console.log(filters);
+                });
+              },
+            ],
+            [
+              "buttonClicked",
+              function (event) {
+                console.log("button", event);
+                window.report.getFilters().then((filters) => {
+                  console.log(filters);
```

Comment:
Could we removed this console log, then, or event handler?

Reply:
Ahhhh  yes sorry, leftover event handlers from testing....forgot to delete when I pivoted lol.  Thank you!